permissions/plugin-ci-with-toad-devops-toolkit.yml

```diff
@@ -1,7 +1,8 @@
----
-name: "ci-with-toad-devops-toolkit"
-github: "jenkinsci/ci-with-toad-devops-toolkit-plugin"
-paths:
-- "io/jenkins/plugins/ci-with-toad-devops-toolkit"
-developers:
-- "hamermike"
+---
+name: "ci-with-toad-devops-toolkit"

```

Comment:
can you please check why this should entirely changed? I suspect a newline format change?